src/models/db/migrations/20240628090906_query_table.ts

```diff
@@ -0,0 +1,39 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  const now = () => knex.fn.now()
+
+  await knex.schema.createTable('queries', (def) => {
+    def.uuid('id').defaultTo(knex.raw('uuid_generate_v4()')).primary()
+    def.uuid('connection_id').notNullable()
+    def.string('company_name').notNullable()
```

Comment:
This is redundent as it's stored in the `connection` table. We should rely on the fk and either do a join or do two queries. Drop the column